container.fc

```diff
@@ -24,6 +25,8 @@
 /var/lib/docker/overlay(/.*)?	gen_context(system_u:object_r:container_share_t,s0)
 /var/lib/docker/overlay2(/.*)?	gen_context(system_u:object_r:container_share_t,s0)
 
+/var/lib/containers/atomic(/.*)?	gen_context(system_u:object_r:svirt_sandbox_file_t,s0)
```

Comment:
THis should be container_share_t.  This is a label that a container process can read/execute but can not write.

svirt_sandbox_file_t has been changed to container_file_t, but this label is writable by all container processes if the MCS label is s0.

Reply:
On which version was this change from 'svirt_sandbox_file_t' to 'container_file_t' introduced?

Reply:
I think it was introduced in the RHEL7.5 release.